hamcrest/Hamcrest.php

```diff
@@ -507,6 +507,32 @@ function notSet($property)
     }
 }
 
+if (!function_exists('anExistingDirectory')) {
```

Comment:
Not sure if you've added this code by hand or automatically, but there is a https://github.com/hamcrest/hamcrest-php/blob/master/generator/run.php script, that scans the code and automatically regenerates this file.

Reply:
I've run this script to generate it.